src/storage.cpp

```diff
@@ -83,6 +81,88 @@ int Storage::NumberOfImages() {
   return images_->length();
 }
 
+// TODO(all): maybe some refactor to get model by name instead index
```

Comment:
you must add points in the end of your comments.
examples: https://google.github.io/styleguide/cppguide.html#Comments